.gitignore

```diff
@@ -0,0 +1,27 @@
+### Windows ###
+# Windows thumbnail cache files
+Thumbs.db
+Thumbs.db:encryptable
+ehthumbs.db
+ehthumbs_vista.db
+
+# Dump file
+*.stackdump
+
+# Folder config file
+[Dd]esktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Windows Installer files
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+###a file should be ignored by git###
+test.md
```

Comment:
Kindly add the `node_modules` folder to the list of ignored on the `.gitignore` file 💯 

You can do it by writing the name of the folder inside the `.gitignore` file, like this 

![image](https://user-images.githubusercontent.com/93630700/197900016-6f4db571-3d10-416b-a127-b938ade86a74.png)

Reply:
Loved that tip, Thank you.